src/js/utils/ui.js

```diff
@@ -318,11 +325,25 @@ const eventRegisters = {
             }
         });
     },
+    keydown(ui) {
+        const keydown = 'keydown';
+        addEventListener(ui, keydown, 'keydown', (e) => {
```

Comment:
does `'keydown'` have to be a string ?
`addEventListener(ui, keydown, keydown, (e) => {`